tests/test_middlewares.py

```diff
@@ -0,0 +1,67 @@
+from django.http.response import HttpResponse
+from django.template.defaultfilters import urlencode
+from django.test import TestCase
+from mock import MagicMock
+
+from rewrite_external_links.middleware import RewriteExternalLinksMiddleware
+
+
+class TestRewriteExternalLinksMiddleware(TestCase):
+    def setUp(self):
+        self.middleware = RewriteExternalLinksMiddleware()
+        self.request = MagicMock()
+        self.content_type = 'text/html'
+        self.link = 'http://example.com'
+        self.content = '<a    href="{}"></a>'.format(self.link).encode()
```

Comment:
It's part of what allow the regex. Ideally we should have a couple more tests to show what is possible.